src/js/_store/modules/categories.js

```diff
@@ -88,6 +102,36 @@ export default {
         commit('failRequest', { message: err.message });
       });
     },
+    getEditCategory({ commit, rootGetters }, editId) {
```

Comment:
[imo]

アクション名や引数名は以下が良いかと思います。

```suggestion
    getCategory({ commit, rootGetters }, id) {
```

今回たまたま更新機能でカテゴリーの詳細を取得する必要があっただけで、他の機能から呼ばれることもあり得ます。
その機能が `Edit` 機能ではなかった場合、このアクションメソッド名に違和感が生じるかと思います。
また、あくまでこのアクションの中で処理している内容は、「カテゴリー詳細 API を実行している」だけですよね。

Reply:
Editの部分を削除しました。